validator/PLANS.md

```diff
@@ -0,0 +1,53 @@
+# Validator checks and features
+
+## Current checks for indicator source data
```

Comment:
Just curious -- have you tried running this on existing imports already?

Specifically I'm wondering if you have a sense of how often this generates false positives (if any) so that the aggressiveness of the validations can be tuned.

Reply:
Yes! I have run this on the USAFacts indicators.

The old Facebook-specific validator had false positives from the check comparing average values of source data vs of the API. I increased the static threshold used in this check to require a larger difference in averages to trigger the error, and haven't yet had problems with it being too sensitive. 

I do often see the "val column can't have any cell smaller than 0" error. Although we expect to see negative values *occasionally*, used to adjusted for previous overreporting, it turns out they are fairly common. These aren't false positives per se, but perhaps given the frequency of the error, we would only want to report it if there were a very large negative value or many small negative values.

Reply:
I'm going to put @chinandrew on surveying the other signals.